src/networking/responses/products-response.ts

```diff
@@ -1,31 +1,35 @@
-export interface Price {
+export interface PriceResponse {
   amount: number;
   currency: string;
 }
 
-export interface PurchaseOptionPrice {
+export interface PurchaseOptionPriceResponse {
   period_duration: string | null;
-  price: Price | null;
+  price: PriceResponse | null;
   cycle_count: number;
 }
 
-export interface PurchaseOption {
+export interface PurchaseOptionResponse {
   id: string;
 }
 
-export interface SubscriptionPurchaseOption extends PurchaseOption {
-  base_price: PurchaseOptionPrice;
-  trial: PurchaseOptionPrice | null;
+export interface SubscriptionPurchaseOptionResponse
+  extends PurchaseOptionResponse {
+  base_price: PurchaseOptionPriceResponse;
```

Comment:
I'm going to rename the type here to `PurchaseOptionPhaseResponse`, and the public interface to `PurchaseOptionPhase`.

As for these fields, I'm wondering if we should rename them to `base_phase` and `trial_phase`. Wdyt? @vicfergar

Reply:
Shipped!